worldedit-core/src/main/java/com/sk89q/worldedit/command/ToolCommands.java

```diff
@@ -191,6 +199,19 @@ public void tree(Player player, LocalSession session,
         setTool(player, session, new TreePlanter(type), "worldedit.tool.tree.equip");
     }
 
+    @Command(
+        name = "stacker",
+        desc = "Block stacker tool"
+    )
+    @CommandPermissions("worldedit.tool.stack")
+    public void stacker(Player player, LocalSession session,
+                      @Arg(desc = "The max range of the stack", def = "10")
+                        int range,
```

Comment:
I'm surprised checkstyle isn't screaming at you for this indentation. It should be 4 spaces in from the `@Arg`. And I think the `@Arg` should be indented to match where `Player` starts, to align the arguments properly. Same applies to `Mask` below.